tools/onnx/sarif/gen_sarif.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# Run this script inside its folder to generate the SARIF python object model files
+# from the SARIF schema.
+# e.g. sh gen_sarif.sh
+#
+# This script requires the jschema_to_python package to be installed.
+# To install it, run:
+#   pip install jschema_to_python
+ROOT=$(pwd)/../../..
```

Comment:
```suggestion
set -e -x
ROOT=$(pwd)/../../..
```
Make sure that CI pipelines (or any other script that calls this one) will fail if the script itself fail

Reply:
This whole script needs to be `shellcheck`'d. Most notably variables are unquoted which can introduce unexpected globbing and splitting.

* https://gist.github.com/abock/620576895d6ae3ed12c9863ac73d283d

Reply:
I updated the script in the last PR in this stack. LMK if that works